README.md

```diff
@@ -17,13 +17,13 @@ This is useful for:
 
 - [Microsoft Visual Studio Code](https://code.visualstudio.com)
 - [Hyper](https://hyper.is/)
-- [Upterm](https://github.com/railsware/upterm)
```

Comment:
I don't think we should be removing this.

Reply:
OK, its archived non working project though, only reason I removed it.